app/controllers/errors_controller.rb

```diff
@@ -6,6 +6,14 @@ class ErrorsController < ActionController::Base
 
   def not_found
     render status: :not_found
+
+    return unless request.referer
+
+    Rollbar.warning(
+      ["RoutingError: No route matches",
+       "[#{request.method}]",
+       "'#{request.original_url}'"].join(" ")
+    )
```

Comment:
Do you know if `Rollbar.warning` automatically includes the stacktrace/request headers/etc.?

I think those will be very useful for eventually debugging dead links.

Reply:
After a quick investigation we've discovered that request headers ARE sent to rollbar. So we'll have all the info we need to debug dead links :partying_face:

Reply:
ah perfect! 💪 can you send a screenshot or something so i know where to look next time?